sympy/functions/elementary/exponential.py

```diff
@@ -687,6 +687,15 @@ def _eval_is_positive(self):
     def _eval_is_zero(self):
         return (self.args[0] - 1).is_zero
 
+    def _eval_is_nonnegative(self):
+        arg = self.args[0]
+        if arg.is_positive:
+            if arg.is_infinite:
+                return True
+            if arg.is_zero:
```

Comment:
Yes, you are right.
Though I was supposed to put `arg.is_nonnegative` , so that it can return False in case of zero.